src/EnergyPlus/SetPointManager.cc

```diff
@@ -5764,26 +6108,7 @@ void DefineOutsideAirSetPointManager::calculate(EnergyPlusData &state)
         SetTempAtOutHigh = this->OutHighSetPt1;
     }
 
-    this->SetPt = CalcSetPoint(OutLowTemp, OutHighTemp, state.dataEnvrn->OutDryBulbTemp, SetTempAtOutLow, SetTempAtOutHigh);
-}
-
-Real64 DefineOutsideAirSetPointManager::CalcSetPoint(
-    Real64 OutLowTemp, Real64 OutHighTemp, Real64 OutDryBulbTemp, Real64 SetTempAtOutLow, Real64 SetTempAtOutHigh)
-{
-    Real64 SetPt;
-    if (OutLowTemp < OutHighTemp) { // && SetTempAtOutLow > SetTempAtOutHigh
-        if (OutDryBulbTemp <= OutLowTemp) {
-            SetPt = SetTempAtOutLow;
-        } else if (OutDryBulbTemp >= OutHighTemp) {
-            SetPt = SetTempAtOutHigh;
-        } else {
-            SetPt = SetTempAtOutLow - ((OutDryBulbTemp - OutLowTemp) / (OutHighTemp - OutLowTemp)) * (SetTempAtOutLow - SetTempAtOutHigh);
-        }
-
-    } else {
-        SetPt = 0.5 * (SetTempAtOutLow + SetTempAtOutHigh);
-    }
-    return SetPt;
```

Comment:
As elaborated on our Design Document, this CalcSetPoint function would now be used in SetpointManager:OutdoorAirReturn, and new objects (SetpointManager:SystemNodeReset:Temperature and SetpointManager:SystemNodeReset:Humidity). Hence, we made the duplicate of this outside DefineOutsideAirSetPointManager and deleted this.

Reply:
But...shouldn't the new set point manager classes have inherited the base class and also used this member method?  Pulling it out into a hanging function when it was already a member method available for all derived classes seems incorrect, but I will concede that I could be missing something.  Please just clarify a bit more, or we could chat about it.